web/src/features/panels/zone/EstimationCard.tsx

```diff
@@ -0,0 +1,203 @@
+import Badge from 'components/Badge';
+import { useState } from 'react';
+import { HiChevronDown, HiChevronUp } from 'react-icons/hi2';
+import { useTranslation } from 'translation/translation';
+import { ZoneDetails } from 'types';
+
+export default function EstimationCard({
+  cardType,
+  estimationMethod,
+  outageMessage,
+}: {
+  cardType: string;
+  estimationMethod: string | undefined;
+  outageMessage: ZoneDetails['zoneMessage'];
+}) {
+  if (cardType == 'outage') {
+    return <OutageCard outageMessage={outageMessage} />;
+  } else if (cardType == 'aggregated') {
+    return <AggregatedCard />;
+  } else if (cardType == 'estimated') {
+    return <EstimatedCard estimationMethod={estimationMethod} />;
```

Comment:
This should probably be a switch case, but let's skip that in this PR.

Reply:
Yes and yes :D